lighthouse-core/gather/gatherers/image-usage.js

```diff
@@ -139,7 +144,12 @@ class ImageUsage extends Gatherer {
       return map;
     }, {});
 
-    return driver.evaluateAsync(`(${collectImageElementInfo.toString()})()`)
+    const expression = `(function() {
+      ${DOMHelpers.getElementsInDocument.toString()};
```

Comment:
took me a moment to figure out what you're doing here.

we should use this library consistently. If this makes the most sense, then in `anchors-with-no-rel-noopener.js` we should use it similar rather than `${fn.toString()}(selector)`